shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -105,8 +131,29 @@
           .hasArg(true)
           .numberOfArgs(1)
           .argName("locality-file")
+          .argName("locality-file")
           .desc("Locality File contains worker localities, each line has a worker locality")
           .build();
+  private static final Option EXCLUDED_LOCALITY_OPTION =
+      Option.builder()
+          .longOpt("excluded-locality")
+          .required(false)
+          .hasArg(true)
+          .numberOfArgs(1)
+          .argName("excluded-locality")
+          .desc("A list of excluded worker locality separated by comma,"
```

Comment:
by comma. It should not be

Reply:
Add the following line to all the locality related options:
Only one of the 'locality', 'locality-file', 'excluded-locality', and 'excluded-locality-file' should be set.